### PR TITLE
feature read.LPJ_input

### PR DESCRIPTION
**Pull request by mppalves**
Adding a new argument to read.LPJ_input (ncells). Bug fixing integer overflow by converting in_nbytes into double

### CODE REVIEW

**Comment by mppalves** [comment]
This addition is necessary to allow the input files to be read either in 59199 or 67420 cells and to fix an integer overflow when multiplying the header of files with daily values. It was tested and has no impact on previous functionality.

File: R/read.LPJ_input.R [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
 #' Reads a LPJmL input file and converts it to a LPJ-object
-#' 
+#'
 #' This function reads in LPJ-input files, using its header information. So
 #' far, tested for landuse input.
-#' 
+#'
 #' @usage read.LPJ_input(file_name, out_years=c("y1995","y2005"),
-#' namesum=FALSE, four_d=FALSE,swap="little")
+#' namesum=FALSE, four_d=FALSE, ncells=59199, swap="little")
```
Comment (tscheypidi) [inline R/read.LPJ_input.R]: please delete the @usage row completely (not needed anymore)

**Review by tscheypidi — COMMENTED** [review]
please remove the @usage section and rerun it with the most recent version of lucode::buildLibrary to create the missing README.md and .zenodo.json files

File: R/read.LPJ_input.R (hunk 1 above)
Comment (mppalves) [reply R/read.LPJ_input.R]: Done!

Comment (mppalves) [reply R/read.LPJ_input.R]: done!

**Review by tscheypidi — APPROVED** [review]

**Pull request merged**